docs/Snapshots.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+---
+<link rel="stylesheet" href="assets/main.css">
+
+## Dafny Documentation snapshots
+
+- [Current development version](https://dafny.org/dafny)
+- [Latest release snapshot](https://dafny.org)
+- [Pre v. 3.8.2](https://dafny.org/V-prehistory)
```

Comment:
I'm not convinced we need this snapshot, since we can have a regular 3.8.2 (or rather 3.9) snapshot very soon.

Reply:
I also slightly prefer the pattern `v3.8.2` rather than `V-3.8.2`

Reply:
It is a placeholder as there are no actual versions yet. I'll delete the V-prehistory when we put a v3.8.2 in place. And I'll use the v3.8.2 syntax at that time.